diff --git a/node_modules/elysia-autoload/dist/index.js b/node_modules/elysia-autoload/dist/index.js
index 88730e9..2ac05f8 100755
--- a/node_modules/elysia-autoload/dist/index.js
+++ b/node_modules/elysia-autoload/dist/index.js
@@ -46,7 +46,7 @@ function autoload(options = {}) {
         const paths = [];
         for await (const path of (0, utils_1.sortByNestedParams)(files)) {
             const fullPath = (0, node_path_1.join)(directoryPath, path);
-            const file = await Promise.resolve(`${fullPath}`).then(s => require(s));
+            const file = await Promise.resolve(`${fullPath}`).then(s => import(s));
             if (!file.default)
                 throw new Error(`${path} doesn't provide default export`);
             const url = (0, utils_1.transformToUrl)(path);
